java/HealthVault Mobile/HVSample/src/main/res/menu/bottom_nav_items.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:id="@+id/menu_home"
+        android:title="@string/menu_home"
+        android:icon="@drawable/home_icon_select"
+        />
+
+    <item
+        android:id="@+id/men_actionplan"
+        android:title="@string/menu_ActionPlan"
```

Comment:
This isn't quite resolved yet. this file was update to be consistent within itself, but the other files in the project don't match.